src/node.cpp

```diff
@@ -1498,7 +1493,7 @@ void LocalNode::setnotseen(int newnotseen)
 }
 
 // set fsid - assume that an existing assignment of the same fsid is no longer current and revoke
-void LocalNode::setfsid(handle newfsid)
+void LocalNode::setfsid(handle newfsid, handlelocalnode_map& fsidnodes)
```

Comment:
This is also necessary for unit testing to avoid having to access the `client` directly.